scripts/calibre-web-nginx.conf

```diff
@@ -1,33 +1,3 @@
-# RELATED: https://github.com/janeczku/calibre-web/wiki/Setup-Reverse-Proxy
-
-# http://box/books
-location {{ calibreweb_url1 }}/ {
-    proxy_set_header        Host            $http_host;
-    proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
-    proxy_set_header        X-Scheme        $scheme;
-    proxy_set_header        X-Script-Name   "{{ calibreweb_url1 }}";
-    proxy_pass http://127.0.0.1:8083;
-}
-
-# http://box/libros
-location {{ calibreweb_url2 }}/ {
-    proxy_set_header        Host            $http_host;
-    proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
-    proxy_set_header        X-Scheme        $scheme;
-    proxy_set_header        X-Script-Name   "{{ calibreweb_url2 }}";
-    proxy_pass http://127.0.0.1:8083;
-}
-
-# http://box/livres
-location {{ calibreweb_url3 }}/ {
-    proxy_set_header        Host            $http_host;
-    proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
-    proxy_set_header        X-Scheme        $scheme;
-    proxy_set_header        X-Script-Name   "{{ calibreweb_url3 }}";
-    proxy_pass http://127.0.0.1:8083;
-}
-
-# http://box/library/calibre-web/<author>/<title>/<webm_or_mp4_video_file>
```

Comment:
Interesting.
i.e. removing {`/books`, `/libros`, `/livres`} for now, if only for testing.

(Preserve comment lines ideally!?)

```
# RELATED: https://github.com/janeczku/calibre-web/wiki/Setup-Reverse-Proxy

# http://box/library/calibre-web/<author>/<title>/<webm_or_mp4_video_file>
```

Reply:
I removed them. We don't need them for testing.

Reply:
Understood:

Please restore both in-line comments to help @EMG70 and others what might be changing!

Reply:
Done